.vscode/launch.json

```diff
@@ -9,6 +9,7 @@
       "request": "attach",
       "port": 5678,
       "host": "localhost",
+      "justMyCode": false,
```

Comment:
Accidental addition?

Reply:
Perhaps should push it into a different commit to make it obvious, but was needed to understand what was happening in the HA energy validation routines :)

Reply:
Yeah I've been enabling it locally as well! Happy to include it if it's deliberate